packages/clerk-js/src/core/resources/User.ts

```diff
@@ -132,6 +132,10 @@ export class User extends BaseResource implements UserResource {
     return new ExternalAccount(json, this.path() + '/external_accounts');
   };
 
+  deleteExternalAccount = async ({ externalAccountId }: { externalAccountId: string }): Promise<void> => {
```

Comment:
🔧 @chanioxaris Since we don't have a `deleteEmailAddress` or `deletePhoneNumber` method on the user object, it's fine to omit it for external accounts too. Instead, users should be able to do `user.externalAccounts[0].destroy()`.

Reply:
Yes you are right, no need to introduce this wrapper on the user resource. Has been removed